libraries/root_generic/options.cpp

```diff
@@ -143,10 +143,10 @@ void vs_options::init() {
             XMLSupport::parse_bool(vs_config->getVariable("graphics", "separatespecularcolor", "false"));
     LockVertexArrays = XMLSupport::parse_bool(vs_config->getVariable("graphics", "LockVertexArrays", "true"));
     fogdetail = XMLSupport::parse_int(vs_config->getVariable("graphics", "fogdetail", "0"));
-    fov = XMLSupport::parse_floatf(vs_config->getVariable("graphics", "fov", "78"));
-    aspect = XMLSupport::parse_floatf(vs_config->getVariable("graphics", "aspect", "1.33"));
-    znear = XMLSupport::parse_floatf(vs_config->getVariable("graphics", "znear", "1"));
-    zfar = XMLSupport::parse_floatf(vs_config->getVariable("graphics", "zfar", "100000"));
+    // fov = XMLSupport::parse_floatf(vs_config->getVariable("graphics", "fov", "78"));
```

Comment:
Why not delete these? I assume they moved to configuration?

Reply:
Yeah, I can probably delete them. They have indeed moved to configuration().

Reply:
Deleted.